### PR TITLE
users: explain layered scratch builds

### PR DESCRIPTION
**Pull request by ktdreyer**
Scratch builds can use other scratch builds in `FROM` instructions. Explain how this works and give an example.

Note, I've not tested this behavior myself, yet. I'm just capturing a discussion with @lcarva here.

### CODE REVIEW

File: docs/users.rst [hunk 1]
```diff
@@ -1092,6 +1092,13 @@ that was pushed to ``rsyslog`` will cause a build failure.
 
 This behavior requires koji integration to be enabled in the OSBS environment.
 
+For scratch builds, OSBS does not resolve a Dockerfile ``FROM`` instruction
```
Comment (rcerven) [inline docs/users.rst:1095]: what do you mean by the 1st sentence???

Comment (lcarva) [reply docs/users.rst:1095]: I think it's meant to capture that OSBS will not ensure there's a corresponding Koji build, i.e. not run koji_parent plugin.
I don't know how to rephrase it to make it easier to understand.

**Review by rcerven — APPROVED** [review]

**Review by MartinBasti — APPROVED** [review]

**Comment by MartinBasti** [comment]
Have you tested it?
